File: SkaiLab.Invoice/ClientApp/src/app/models/permission.ts
export class Permission{
    readWritePurchaseSale:boolean;
    readPurchaseSale:boolean;
    approvaPayPurchaseSale:boolean;
    manageOrganisactionSetting:boolean;
    manageAndInviteUser:boolean;
    payroll:boolean;
    report:boolean;
    constructor(){
        this.readWritePurchaseSale=false;
        this.readPurchaseSale=false;
        this.approvaPayPurchaseSale=false;
        this.manageOrganisactionSetting=false;
        this.manageAndInviteUser=false;
        this.payroll=false;
        this.report=false;
    }
}

File: SkaiLab.Invoice.Models/Permission.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace SkaiLab.Invoice.Models
{
    public class Permission
    {
        public bool ReadWritePurchaseSale { get; set; }
        public bool ReadPurchaseSale { get; set; }
        public bool ApprovaPayPurchaseSale { get; set; }
        public bool ManageOrganisactionSetting { get; set; }
        public bool ManageAndInviteUser { get; set; }
        public bool Payroll { get; set; }
        public bool Report { get; set; }
    }
}
